test/commons/dom/is-visible.js

```diff
@@ -174,6 +188,21 @@ describe('dom.isVisible', function() {
       assert.isFalse(axe.commons.dom.isVisible(el));
     });
 
+    it('should detect when clip is not applied because of positioning on parent', function() {
```

Comment:
Why are you repeating tests?

Reply:
I'm not - this is when clip is not applied because of positioning on the parent, the previous test is positioning on the element itself. Down below, it's testing when the screenreader option is set using these same two cases.